tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -57,3 +57,6 @@ add_msr_test (cache_shelltests "${CMAKE_SOURCE_DIR}/src/plugins/cache/shelltests
 # Only works with super user privileges, since it writes to `/etc/hosts`:
 # add_msr_test (tutorial_mount "${CMAKE_SOURCE_DIR}/doc/tutorials/mount.md")
 # ~~~
+foreach (PLUGIN ${ADDED_DIRECTORIES})
+       add_msr_test (${PLUGIN} "${CMAKE_SOURCE_DIR}/src/plugins/${PLUGIN}/README.md")
+endforeach (PLUGIN)
```

Comment:
This can be commented out later on because `add_msr_test` is already called within TEST_README, which is the proper way to include the tutorial wrapper tests for **plugins**.

```suggestion

# ==========
# = Tutorials =
# ==========
 
# tutorials get added by TEST_README in add_plugin
# to test if all tutorials work (also the ones not yet included) uncomment:

#foreach (PLUGIN ${ADDED_DIRECTORIES})
#       add_msr_test (${PLUGIN} "${CMAKE_SOURCE_DIR}/src/plugins/${PLUGIN}/README.md")
#endforeach (PLUGIN)
```